back/node_watcher/kubernetes/nomidotwatcher-job.yaml

```diff
@@ -1,30 +1,38 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: batch/v1
+kind: Job
 metadata:
   name: nomidotwatcher
   namespace: nodewatcher-staging
   labels:
     stage: staging
     name: nomidotwatcher
     app: nomidotwatcher
+    job-name: nomidotwatcher
 spec:
+  manualSelector: true
   selector:
     matchLabels:
+      stage: staging
+      name: nomidotwatcher
       app: nomidotwatcher
-  replicas: 1
-  strategy:
-    type: RollingUpdate
+      job-name: nomidotwatcher
   template:
     metadata:
       labels:
         stage: staging
         name: nomidotwatcher
         app: nomidotwatcher
+        job-name: nomidotwatcher
     spec:
+      restartPolicy: Never
       containers:
         - name: nomidotwatcher
-          image: eu.gcr.io/test-installations-222013/nomidot_watcher@sha256:73f32773eb4e397e9f4cc7028945750f8f1116c7e751179120300a18413797ee
+          image: eu.gcr.io/test-installations-222013/nomidot_watcher@sha256:22592d41b85880683b595de06f3b399eef3b42750baf02f6afc73e85130152ea
+          imagePullPolicy: Always
           env:
             - name: PRISMA_ENDPOINT
               value: http://10.0.9.43:4466
+            - name: START_FROM
+              value: '37921'
```

Comment:
Are you trying to spawn multiple nomidot watchers? Why start at this block?

Reply:
there was a bug with proposals around there, so process would exit, pod restarts from where the last job started (i had hardcoded for the previous version). It's a workaround just to get a full sync at least once so we have one to work with while figuring out a more sustainable way https://github.com/paritytech/Nomidot/issues/109